app/routes.py

```diff
@@ -1 +1,135 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from sqlalchemy import desc, asc
+import datetime as dt
+import os
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
```

Comment:
We don't need to import `dotenv` again in the routes. We executed these lines in the `__init__.py` file. Inside that file, we imported the `routes.py` files. That means that the `routes` get the benefit of the `__init__.py` file's `load_dotenv()`

 ```suggestion
import requests
```